### ISSUE TITLE
Handle disconnected islands for python solver

### ISSUE THREAD

**Issue by vkgnosis** [gnosis/gp-v2-services#253]
Driver has to split up orders on disconnected tokens into separate instances.

**Comment by fleupold** [comment]
Is this still the case @marcovc @twalth3r? Or is the MIP solver now handling disconnected islands as well?

**Comment by marcovc** [comment]
No, not yet, since price estimation requires that things are connected. But as far as I know, this is already implemented in the driver: https://github.com/gnosis/gp-v2-services/blob/main/solver/src/http_solver.rs#L239-L265
